snuba/subscriptions/data.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from typing import List, Tuple
+
+from snuba.datasets.dataset import Dataset
+from snuba.views import validate_request_content
+from snuba.query.types import Condition
+from snuba.request import Request
+from snuba.request.schema import RequestSchema
+
+
+@dataclass(frozen=True)
+class Subscription:
+    """
+    Represents the state of a subscription.
+    """
+    id: str
```

Comment:
Oops, yeah — you're right. The ID also needs to include the partition key so that operations after create can be routed to the correct project. This might tie in a bit with this comment then too: https://github.com/getsentry/snuba/pull/618#discussion_r356214864